packages/nodes-base/nodes/Form/v2/FormTriggerV2.node.ts

```diff
@@ -94,6 +94,29 @@ const descriptionV2: INodeTypeDescription = {
 				respondWithOptions,
 			],
 		},
+		{
+			displayName: 'Options',
+			name: 'options',
+			type: 'collection',
+			placeholder: 'Add Option',
+			default: {},
+			displayOptions: {
+				show: {
+					responseMode: ['responseNode'],
+				},
+			},
+			options: [
+				{
+					// eslint-disable-next-line n8n-nodes-base/node-param-display-name-miscased
+					displayName: 'Append n8n Attribution',
+					name: 'appendAttribution',
+					type: 'boolean',
+					default: true,
+					description:
+						'Whether to include the link “Form automated with n8n” at the bottom of the form',
+				},
+			],
+		},
```

Comment:
I would suggest not to add a new options but to modify existing as follow to reduce duplication
```
		{
			displayName: 'Options',
			name: 'options',
			type: 'collection',
			placeholder: 'Add Option',
			default: {},
			options: [
				{
					// eslint-disable-next-line n8n-nodes-base/node-param-display-name-miscased
					displayName: 'Append n8n Attribution',
					name: 'appendAttribution',
					type: 'boolean',
					default: true,
					description:
						'Whether to include the link “Form automated with n8n” at the bottom of the form',
				},
				{
					...respondWithOptions,
					displayOptions: {
						hide: {
							'/responseMode': ['responseNode'],
						},
					},
				},
			],
		},
```

Reply:
That makes sense, I had issues with display options in options in the past so didn't do it but this is working thanks.